### DISCUSSION TITLE
Qualquer um pode participar ou precisa ser pré selecionado?

### DISCUSSION THREAD

**Discussion by caematos** [bacen/pilotord-kit-onboarding#52] [Q&A] [answered]
Para integrar com o ambiente de testes do Real digital é necessário ser selecionado ou qualquer um pode começar a testar?

Obrigado.

**Comment by BrunooMoniz** [comment, marked as answer]
@caematos para o teste do Piloto DREX foram s[elecionadas 16 consórcios](https://www.bcb.gov.br/estabilidadefinanceira/piloto-drex#:~:text=Piloto%20Drex%20%E2%80%93%20participantes%20selecionados).

**Comment by aldenio** [comment]
Prezado @caematos,

O comentário do @BrunooMoniz está correto. Existe um grupo de instituições selecionadas para participar do piloto do DREX.

obs: caso sua dúvida tenha sido respondida, favor encerrar a discussão.
